rackcity/views/pdu_views.py

```diff
@@ -332,6 +332,7 @@ def power_availability(request):
         if response:
             return response
         assets, assets_cp = get_assets_for_cp(change_plan.id)
+        assets = assets.filter(rack=rack.id)
```

Comment:
will approve but want to flag this just in case / I'm not sure - why are you also not filtering assets_cp by rack?

Reply:
oh thats a bug
fixigin